packages/eslint-plugin/docs/rules/no-new-in-loop.md

```diff
@@ -0,0 +1,25 @@
+Instantiating new objects in the frame loop wastes large amounts of memory causing the garbage collector to do more work than needed.
```

Comment:
Let me know if there should be any wording changed.

Reply:
More specifically, it allocates or leaks memory very quickly. It's particularly bad for large CPU containers like three classes or any GPU resource since there is no (reliable) GC there.

Reply:
Thanks will update it.